glitter/publisher/apps.py

```diff
@@ -0,0 +1,9 @@
+from __future__ import unicode_literals
+
+from django.apps import AppConfig
+
+
+class PublisherConfig(AppConfig):
```

Comment:
Is there any specific reason why this define? As I think I haven't defined `apps.py` as I felt no need for it.